Data/Row/Variants.hs

```diff
@@ -161,6 +162,16 @@ multiTrial (OneOf l x) = if l `elem` labels @(y .\\ x) @Unconstrained1 then Righ
 view :: KnownSymbol l => Label l -> Var r -> Maybe (r .! l)
 view = flip trial'
 
+-- | Split a variant into two sub-variants.
+split :: forall s r. (Forall s Unconstrained1, Subset s r)
```

Comment:
I think with the way that `Subset` is written, it is guaranteed that `s` must have all unique labels, but it doesn't hurt to add that as a type constraint.  My preference would be to replace `Forall s Unconstrained1` with `WellBehaved s`.